Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM php:8-cli
+FROM php:7.4-cli
```

Comment:
proč 7.4 jestli kvůli csv-options tak bysme měli vydat novů verzi nebo by šlo ignorovat ten requirement

Reply:
Zatim to nejspis nepujde... Je tam totiz zavislost:
- `php-db-import-export`  - php8 ready
  - `php-csv-options`  - php8 ready
  - `php-csv-db-import` - `^7.1`
    - `phpunit/dbunit` - `^7.1` + [no longer maintained](https://github.com/sebastianbergmann/dbunit/issues/217)

Muzeme to forknout a upravit (nejaky takovy forky uz jsou)...